app/src/main/java/com/nsoft/github/data/remote/calls/ApiCall.kt

```diff
@@ -67,6 +69,133 @@ data class QueriedApiCall<NetworkParams : RequestParams, out DomainClass : Respo
     }
 }
 
+/**
+ * Base class for OUR wrapped version of the PATH Retrofit call which also contains it's
+ * [RequestAdapter] and [ResponseAdapter] bundled with it so that DI can do most of the magic.
+ *
+ * This class should be used as a "wrapper" around query-based API calls such as
+ * ```
+ * https://backend.com/endpoint/{queryParam1}/{queryParam2}/...
+ * ```
+ *
+ * **IMPORTANT NOTE:** to comply with the rest of the [ApiCall] definition, this call will take it's
+ * path (see [Path]) parameters from the **third** map ([QueryMap]) returned by the [RequestAdapter.convert],
```

Comment:
oh oh oh, instead of using queryMap.values(), lets order them in the map by using predefined keys! like for LITERAL_URL

comment is here to also update the javadoc

Reply:
done done

Reply:
resolving.